diagnostic.md

```diff
@@ -3,67 +3,71 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
+$('a')
+$('h3')
+$('img')
+$('p')
 
-<!-- your answer ends here -->
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$('ol').children();
+$('li').siblings();
 
 ## Question 3
 
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$('.listerItem').parentUtil('form');
```

Comment:
this is fine, but you spelled `parentsUntil` wrong... https://api.jquery.com/parentsUntil/